metadata/core.xml

```diff
@@ -7,34 +7,12 @@
 		<option name="plugins" type="string">
 			<_short>Plugins</_short>
 			<_long>Loads the specified plugins, space-separated list.</_long>
-			<default>
-				alpha
-				animate
-				autostart
-				command
-				decoration
-				expo
-				fast-switcher
-				fisheye
-				grid
-				idle
-				invert
-				move
-				oswitch
-				place
-				resize
-				switcher
-				vswitch
-				window-rules
-				wobbly
-				wrot
-				zoom
-			</default>
+			<default>alpha animate autostart command decoration expo fast-switcher fisheye grid idle invert move oswitch place resize switcher vswitch window-rules wobbly wrot zoom</default>
 		</option>
 		<option name="close_top_view" type="activator">
 			<_short>Close view</_short>
 			<_long>Closes the currently focused window with the specified key.</_long>
-			<default>&lt;super&gt; KEY_Q | &lt;alt&gt; KEY_F4</default>
+			<default>&lt;super&gt; KEY_W | &lt;super&gt; BTN_MIDDLE | &lt;alt&gt; KEY_F4</default>
```

Comment:
super+BTN_MIDDLE might happen by mistake when trying to do another binding. This is not likely to happen with alt+f4. super+KEY_W seems strange to me, KEY_Q makes more sense here.